spring-graphql-api/src/main/java/com/okta/developer/demo/SpringBootApiApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.okta.developer.demo;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class SpringBootApiApplication {
```

Comment:
Did this class get renamed in the blog post as well?

Reply:
I checked and logs where updated